product_docs/docs/eprs/7/10_appendix/03_miscellaneous_xdb_processing_topics/01_publications_and_subscriptions_server_conf_options/snapshot_connection_retry.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Snapshot replication connection retry"
+---
+
+When a connection failure with the subscription database occurs during a snapshot replication, 
+the Replication Server automatically attempts to reconnect to it. 
+The goal is that the snapshot replication completes without skipping any tables. When the connection is reestablished, 
+Replication Server restarts or resumes the snapshot replication procedure for the table that was being migrated when the connection was lost and then performs the data copy for the remaining tables.
+
+!!!note Scope and limitations
+
+   Database scope: The connection retry capability allows Replication Server to reconnect to the target/subscription database. 
+   Retry attempts for connection issues with the source/publication database are not supported.
+
+   Replication scope: This capability allows Replication Server to retry replicating data. 
+   Issues with schema replication are not supported.
+!!!   
+
+You can specify several connection retry options:
+
+- `-snapshotConnRetryCount [<connection_attempts>]`
+
+Use the `-snapshotConnRetryCount` option to specify the number of retry attempts to perform in case the subscription database connection fails in a Snapshot session. 
+The `[<connection_attempts>]` value must be a number between 0 and 50, the default is 3 retry attempts. 
+
+- `-snapshotConnRetryInterval [<seconds>]`
+
+Use the `-snapshotConnRetryInterval` option to specify the seconds to wait before each subsequent reconnection attempt 
+in case the connection to the subscription database fails in a Snapshot session. 
+The `[<seconds>]` value must be a number between 0 and 900, the default is 30 seconds. 
+
+- `-snapshotAbortOnConnFailure [true/false]`
+
+Specify if to abort the snapshot replication if all the reconnection attempts fail. 
+
+Set the `-snapshotAbortOnConnFailure` to `false`, to skip replicating the failed table and proceed to the next table. 
+The default is `true`, which aborts the Snapshot session if the connection fails after the the specified `-snapshotAbortOnConnFailure` threshold.
```

Comment:
Replace "the the specified -snapshotAbortOnConnFailure threshold" with "the specified -snapshotConnRetryCount threshold"